docs/start/install.mdx

```diff
@@ -197,7 +197,8 @@ Additionally, Qiskit only supports the CPython implementation of the Python lang
 
   - Linux x86_64 (distributions compatible with the [manylinux 2014](https://www.python.org/dev/peps/pep-0599/) packaging specification).
   - macOS x86_64 (10.12 or later)
-  - Windows 64 bit
+  - macOS ARM64 (11.0 or newer)
+  - Windows 64 bit (Windows 10 and later supported)
```

Comment:
```suggestion
  - Windows 64 bit (Windows 10 and later supported)
```
I think 64-bit should be hyphenated, unless it's standard in this OS context??

Reply:
It looks like they're hyphenated in the windows official documentation: https://support.microsoft.com/en-us/windows/32-bit-and-64-bit-windows-frequently-asked-questions-c6ca9541-8dce-4d48-0415-94a3faa2e13d so let's make the change

```suggestion
  - Windows 64-bit (Windows 10 and later supported)
```